src/swarm/neo/client/mixins/RequestCore.d

```diff
@@ -235,10 +304,10 @@ public template RequestCore ( RequestType request_type_, ubyte request_code,
 
     ***************************************************************************/
 
-    private static void notify ( ref UserSpecifiedParams params,
+    private static void notify ( UserSpecifiedParams* params,
```

Comment:
Maybe in the next major we could make this `Const!(UserSpecifiedParams)*` params? So that would make a breaking change, since the protos must update their `Notifier` definitions, to accept `Args` by const, but users would just have the read-only view of their args in the notifiers.